### ISSUE TITLE
[New] Guidance on Formatting Tattoo/Piercing Fields

### ISSUE THREAD

**Issue by AdultSun** [stashapp/StashDB-Docs#9]
We haven't had a clear consensus on how we want to format tattoos and piercings for performers. We need to decide:

1. If we want to standardize our "locations" somehow. And if so, to what? (I know Echo has some ideas on how to do this.)
2. Do we want to standardize capitalization? Specifically, capitalize the first word of just the location, first word of location and description, every word of location and description, or no caps at all?

I'm pretty sure the most common method right now is simply to copy-paste phrases from IAFD or FreeOnes into the right fields unchanged.

**Comment by AdultSun** [comment]
Also, there was an edit I just saw in the queue that prompted me to make this issue. One of the tattoos was so large it was originally listed as "Left Shoulder And Upper Arm (Dragon Design)" and the edit split it into "Left shoulder (Dragon Design)" and "Upper Arm (Dragon Design). If we try to standardize our tattoo locations, how will we handle single tattoos that are so large they cross over multiple locations like this?

**Comment by echo6ix** [comment]
#### (1) Standardized location schema
- Advantages
  - Presentation consistency
  - Data consistency
  - Terms can be easily changed with a bulk edit
  - Ease of use with predefined terms might encourage data input for people who were unsure what to type?
- Disadvantages:
  - With predefined terms, copy-and-paste user input not possible
  - Might take longer to input data?

**A**. Without the aid of a GUI and using pure manual data entry it is a big ask for contributors to follow a location schema. It's not worth pursing imo.

**B**. Using a GUI similar to this https://www.thenude.com/index.php?page=search&action=index#search-tab-tattoos (you must click the _tattoos search_ tab) where the user clicks a location on the image and enters a description of the body mod (if any)
- Forces compliance because users can only enter the location of body mods by clicking a location on the GUI
- Results in standardization of location string data
- Restrictive when covering broad areas without adding more complexity for selecting multiple areas.

**C**. Use a dropdown input form with predefined terms
- Implementation would be less complex than (B) and can include broad and specific regions
- Forces compliance because users can only enter the predefined values
- Results in standardization of location string data
- How would the values be sorted?
  - Sorting anatomically could be very intuitive

D. Location field terms
To address body mods that cover multiple ranges perhaps the best approach here might be to start with and include broad anatomical terms. This first iteration is by no means meant to be completely inclusive or accurately categorized.

- head
  - face
    - forehead
    - eyebrows
      - left eyebrow
      - right eyebrow
    - left temple
    - right temple
    - nose
      - left nostril
      - right nostril
    - lips
      - upper lip
      - bottom lip
    - chin
    - cheeks
      - left cheek
      - right cheek
   - ears
     - left ear
     - right ear
  - neck (or neck front)
  - nape (or neck back)
- shoulders
  - left shoulder front
  - left shoulder back
  - right shoulder front
  - right shoulder back
- right arm
  - right upper arm front
  - right upper arm back
  - right middle arm front
  - right middle arm back
  - right lower arm front
  - right lower arm back
  - right inside wrist 
  - right outside wrist
- right hand
  - right hand thumb
  - right hand index finger
  - right hand middle finger
  - right hand ring finger
  - right hand little finger
- left arm
  - left upper arm front
  - left upper arm back
  - left middle arm front
  - left middle arm back
  - left lower arm front
  - left lower arm back
  - left inside wrist 
  - left outside wrist
- left hand
  - left hand thumb
  - left hand index finger
  - left hand middle finger
  - left hand ring finger
  - left hand little finger
- front torso
  - left flank  
  - left upper chest
    - left breast
  - right flank
  - right upper chest
    - right breast 
  - abdomen
    - left abdomen
    - right abdomen
    - navel
  - right front hip
  - left front hip
- pubic region
    - genitals
      - vulva
      - clitoris
      - penis
      - scrotum
- back
  - spine
  - left back
  - right back
  - shoulder blades
    - left shoulder blade
    - right shoulder blade
  - middle black
    - left middle back
    - right middle back
  - lower back
    - left lower back
    - right lower back
  - buttock
    - left buttock
    - right buttock 
- legs
  - right leg
  - left leg
  - thighs
    - right front thigh
    - right back thigh
    - left front thigh
    - left back thigh
  - right lower leg
  - left lower leg
  - right calf
  - left calf
  - ankles
    - left ankle
    - right ankle
- feet
  - left foot
  - right foot

#### (2) Case
_Location field:_ All lowercase.

_Description field_: All lowercase.
I personally have been capitalizing proper nouns out of habit, but I'm not sure if the capitalizing of proper nouns matters here.

Reasons:
- Lowercase is the simplest algorithm to follow here
- Stash and StashDB can style the data with CSS or JavaScript if it's desired

**Comment by PhuriousGeorge** [comment]
I agree with @echo6ix (too long; read most) regarding location.  Regarding capitalization, this discussion has occurred recently on tags as well.  Whatever stance is taken should for the same reasons be utilized uniformly across all metadata/fields excluding of course Performer names, Studios, Series and Scene titles.  Those should be styled in accordance with the source.
